packages/react/src/auto/polaris/inputs/PolarisAutoJSONInput.tsx

```diff
@@ -12,21 +12,17 @@ export const PolarisAutoJSONInput = (
     control?: Control<any>;
   } & Partial<TextFieldProps>
```

Comment:
Perhaps `Partial<Omit<TextFieldProps, "onChange">>` for the same reason as the MUI variant